src/mo_frontend/typing.ml

```diff
@@ -729,6 +729,10 @@ let infer_class_cap env obj_sort (tbs : T.bind list) cs =
   | _ ->
     C.NullCap, tbs, cs
 
+let find_field (ef : exp_field) (fts : T.field list) = 
+  let id = ef.it.id.it in
+  List.find_opt T.(fun ft -> ft.lab = id && not (is_typ ft.typ)) fts
```

Comment:
`exp_field` is always value-level, yes? So we need to skip the `type <id> = <typ>` T.fields.
`normalize` not necessary, right?

Reply:
There should be a Type.find_val_field_opt or similar that does this for I think.